src/main/scala/io/iohk/ethereum/vm/Memory.scala

```diff
@@ -70,6 +70,21 @@ class Memory private(private val underlying: ByteString) {
       (newUnderlying.slice(start, end), new Memory(newUnderlying))
     }
 
+  /** This function will expand the Memory size as if storing data given the `offset` and `size`.
+    * If the memory is already initialised at that region it will not be modified, otherwise it will be filled with
+    * zeroes.
+    * This is required to satisfy memory expansion semantics for *CALL* opcodes.
+    */
+  def expand(offset: UInt256, size: UInt256): Memory = {
+    val totalSize = (offset + size).toInt
+    if (this.size >= totalSize || size.isZero)
```

Comment:
Lol..no, it's eq 223, my bad.

I feel stupid but, where do you get that revision numbers? Mine only says 
![screenshot from 2017-04-26 07-47-15](https://cloud.githubusercontent.com/assets/3750504/25430840/9ddf36d0-2a54-11e7-83a3-6a68b544f4fd.png)